src/Microsoft.Azure.WebJobs.Host/Bindings/DefaultAttributeInvokerDescriptor.cs

```diff
@@ -47,15 +37,19 @@ public TAttribute FromInvokeString(string invokeString)
             }
         }
 
-        public string ToInvokeString()
+        // Default implementation of IAttributeInvokeDescriptor which can be operate on any attribute and 
+        // follows basic conventions with a [ResolveProperty]. 
```

Comment:
good catch

---
In reply to: [62568238](https://github.com/Azure/azure-webjobs-sdk/pull/717#discussion_r62568238) [](ancestors = 62568238)